api_devauth.go

```diff
@@ -254,11 +254,7 @@ func (d *DevAuthHandler) UpdateDeviceStatusHandler(w rest.ResponseWriter, r *res
 		return
 	}
 
-	devurl := utils.BuildURL(r, uriDevice, map[string]string{
-		":id": devid,
-	})
-
-	w.Header().Add("Location", devurl.String())
+	w.Header().Add("Location", "../../devices/"+devid)
```

Comment:
right, of course this form can be shortened, and I don't see why not, trying it out atm.